infrastructure/kctf-samples/apache-php/k8s/deployment/autoscaling.yaml

```diff
@@ -2,5 +2,6 @@ apiVersion: "autoscaling/v1"
 kind: "HorizontalPodAutoscaler"
 metadata:
   name: "chal"
-#spec:
-#  maxReplicas: 3
+spec:
+  minReplicas: 2
```

Comment:
this is to test if the replication works. k8s usually schedules this in two different nodes, but in theory they could get scheduled in the same one